analyzers/tests/SonarAnalyzer.UnitTest/AnalysisContext/SonarAnalysisContextTest.Register.cs

```diff
@@ -188,6 +190,30 @@ public void SonarCompilationStartAnalysisContext_RegistSymbol_ReportsIssue()
         startContext.RaisedDiagnostic.Should().NotBeNull().And.BeSameAs(diagnostic);
     }
 
+#if NET
+
+    [TestMethod]
+    public void RaisedIssue_WithinRazorGeneratedCode_ShouldNotBeReported()
+    {
+        using var scope = new EnvironmentVariableScope(false) { EnableRazorAnalysis = true };
+        new VerifierBuilder()
+            .AddAnalyzer(() => new DummyAnalyzerRazorGeneratedCode())
```

Comment:
There is no need to create a new dummy analyzer. you can also you DummyCS to assert the functionality.
```suggestion
            .AddAnalyzer(() => new DummyCS())
```